src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2310,6 +2311,7 @@ ostree_sysroot_write_deployments_with_options (OstreeSysroot     *self,
   gboolean bootloader_is_atomic = FALSE;
   SyncStats syncstats = { 0, };
   g_autoptr(OstreeBootloader) bootloader = NULL;
+  g_autofree const char *bootloader_config = NULL;
```

Comment:
That should just be `const char*`, no?

Reply:
Ahh, yes, shouldn't be freeing that here.